test/sourcemap.coffee

```diff
@@ -84,21 +84,21 @@ test "node --enable-source-map built in stack trace mapping", ->
         equal column, 9 # Mapped source column
 
         resolve()
-      catch e
-        reject(e)
+      catch exception
+        reject exception
 
-unless process.version.slice(1, 3) is "12"
+if Number(process.version.slice(1, 3)) >= 14
```

Comment:
```suggestion
if Number(process.versions.node.split('.')[0]) >= 14
```

I know your version won’t break until Node v100, but why not make it handle any version? And handle if they ever remove the leading `v`.